src/Admin/Controllers/OrganizationsController.cs

```diff
@@ -208,13 +212,23 @@ public async Task<IActionResult> Edit(Guid id, OrganizationEditModel model)
             throw new BadRequestException("Plan does not support Secrets Manager");
         }
 
-        await _organizationRepository.ReplaceAsync(organization);
-        await _applicationCacheService.UpsertOrganizationAbilityAsync(organization);
-        await _referenceEventService.RaiseEventAsync(new ReferenceEvent(ReferenceEventType.OrganizationEditedByAdmin, organization, _currentContext)
+        try
         {
-            EventRaisedByUser = _userService.GetUserName(User),
-            SalesAssistedTrialStarted = model.SalesAssistedTrialStarted,
-        });
+            await _stripeSyncService.UpdateCustomerEmailAddress(organization.GatewayCustomerId, organization.BillingEmail);
```

Comment:
👍 Nice, I wasn't aware that [IStripeSyncService](https://github.com/bitwarden/server/blob/90d600db9d55c8bf91dbfde286301f4097a71fc0/src/Core/Services/IStripeSyncService.cs#L3) already did this. Good find!